src/components/displayQuote.jsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react';
+import Loader from './Loader';
+
+const DisplayQuote = () => {
+  const [quote, setQuote] = useState(null);
+  const [error, setError] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchQuote = async () => {
+      try {
+        const API_KEY = 'https://api.api-ninjas.com/v1/quotes?category=funny';
```

Comment:
- [ ] [OPTIONAL] The highlighted line of code is actually a URL to fetch funny quotes and not the API key. Your naming of variables may not necessarily affect code execution, but it impacts on how people read and understand your code. Please consider renaming this `API_KEY` variable to avoid confusion 😎